### PR TITLE
Add protocols section to admin_peers (#1580)

### PR DESCRIPTION
**Pull request by shemnon**
Add the protocols section to admin_peers.  This involved plumbing
EthPeers through where we were previously using the P2PNetwork to get
our data.  Hence most of the PR is DI refactoring.

Signed-off-by: Danno Ferrin <danno.ferrin@gmail.com>

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by ryanschneider — APPROVED** [review]
lgtm!  Thanks for the super quick turnaround on #1580 !

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 20.10.2
+* Add a protocols section to `admin_peers` to provide info about peer health. [\#1562](https://github.com/hyperledger/besu/pull/1562)
```
Comment (macfarla) [inline CHANGELOG.md:4]: I think you mean 1582

Comment (macfarla) [reply CHANGELOG.md:4]: and should this be under the Additions and Improvements heading?

**Review by macfarla — APPROVED** [review]
LGTM. just the changelog needs a small fix

File: CHANGELOG.md (hunk 1 above)
Comment (shemnon) [reply CHANGELOG.md]: Done.

**Pull request merged**